CONTRIBUTING.md

```diff
@@ -28,44 +28,107 @@ This step is important because there may be aspects you didn't think about, or t
 
 Bocadillo has a single repository for both the `bocadillo` package and its documentation.
 
-### Setting up the repo
+### Configuring the development environment
 
-Here's how to get your copy of the repo ready for development:
+All development is done on ubuntu 18.04 LTS. Assuming you have just ubuntu fresh, this section will outline the steps to installing dependencies and tools required for development on the project.  
```

Comment:
Ubuntu version is 18.04.1. Please update.

Reply:
version updated